sharing_portal/templates/sharing_portal/share.html

```diff
@@ -67,6 +68,18 @@ <h2 class="artifactTitle">Edit sharing settings</h2>
             </p>
             {% bootstrap_field share_form.projects show_label=False %}
           </fieldset>
+
+          <p class="help-block">
+            A person without an active Chameleon allocation is unable to launch
+            and reproduce your artifact. If you enable reproducability requests,
+            users can send requests, and if granted, they will be given
+            access to a private Chameleon project for the given number of hours.
```

Comment:
I think maybe the information most relevant to the PI is:

"if granted, they will be given a small temporary allocation, which will be tied to your project's allocation. While usage by reproducing users will not count towards your project's main allocation, the PI is nonetheless accountable for their responsible usage of Chameleon resources. All reproducibility requests therefore require PI approval."

Or maybe something simpler.

Reply:
Updated.